### PR TITLE
Fix issues and get ICEES KG ready for September Relay

### PR DESCRIPTION
**Pull request by maximusunc**

### CODE REVIEW

**Review by karafecho — CHANGES_REQUESTED** [review]
I believe these changes look fine EXCEPT that the URL for the Exposures Provider ICEES wiki page is incorrect and should be https://github.com/NCATSTranslator/Translator-All/wiki/Exposures-Provider---ICEES (three hyphens before "ICEES"). I think this might have been changed. Either that or it was incorrectly listed on the Exposures Provider wiki page, although I'm pretty sure I checked all of the URLs on that page before saving changes. Anyway, we need to correct this on the edge properties. Thanks!

**Comment by maximusunc** [comment]
Is there are reason for the three hyphens for the wiki page? If not, it would be much easier to update the wiki page url to only include one (those hyphens are just spaces in the title of the page, so we could easily just delete the extra spaces)

File: icees_kg/utils/jsons_to_csv.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""This is a throwaway script that converts jsonl files to csvs."""
```
Comment (kennethmorton) [inline icees_kg/utils/jsons_to_csv.py]: I think you forgot to throw it away!

Comment (kennethmorton) [reply icees_kg/utils/jsons_to_csv.py]: We should probably clean up (remove?) the comment blocks.

File: icees_kg/utils/parse_jsonlines.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+"""This is a throwaway script that parses jsonl files for various purposes."""
```
Comment (kennethmorton) [inline icees_kg/utils/parse_jsonlines.py]: Fix header?

**Review by kennethmorton — DISMISSED** [review]
This all looks good just a few minor things, but I'll add approval now.

**Comment by karafecho** [comment]
I fixed the header on the wiki page: https://github.com/NCATSTranslator/Translator-All/wiki/Exposures-Provider-ICEES. Don't know why I didn't think of that!

Question: For consistency with the other Exposures Provider wiki pages, do you think I should re-title the ICEES+/ICEES KG terms & conditions of service wiki page by adding "Exposures Provider"?

File: icees_kg/utils/jsons_to_csv.py (hunk 1 above)
Comment (maximusunc) [reply icees_kg/utils/jsons_to_csv.py]: I removed all these extra one-off scripts. They shouldn't be needed once the pre-compute script is finalized.

**Review by karafecho — APPROVED** [review]
Looks good to me!

**Review by kennethmorton — APPROVED** [review]

**Pull request merged**